src/navigator/TabNavigator.tsx

```diff
@@ -38,7 +38,7 @@ function getHeaderTitle(route: RouteProp<any, any>, defaultValue: string) {
         case Screens.CommunityManager:
             return i18n.t('generic.manage');
         case Screens.Communities:
-            return i18n.t('generic.communities');
+            return null;
```

Comment:
The catch here is that we redefined all the UI but kept the navigations as it was in order to not refactor everything at once. Without this hack, the title is inherit as "TabNavigator" leading us to have to use an approach like this:

<Navigator.Screen
            name=" " // <<-- with a space as screen name.
            component={TabNavigator}
        />

I choose the first one because it's more explicit and ease to understand.